spec/requests/querying_spec.rb

```diff
@@ -966,7 +966,7 @@ def create_vms_by_name(names)
   describe 'OPTIONS /api/vms' do
     it 'returns the options information' do
       options(api_vms_url)
-      expect_options_results(:vms)
+      expect_options_results(:vms, {})
```

Comment:
Previously, this method defaulted to `{}` so we need to change the one caller that wasn't providing a hash and expected it to be defaulted.